contrib/babelfishpg_common/src/collation.c

```diff
@@ -1084,9 +1121,15 @@ get_server_collation_oid_internal(bool missingOk)
 
 Oid BABELFISH_CLUSTER_COLLATION_OID()
 {
-
 	if (sql_dialect == SQL_DIALECT_TSQL)
```

Comment:
We are mostly trying to fix the default collation of Babelfish data types which would sub sequently fix sys objects dependent on those data types because collation would bubbled up from default collation of data types. 

So given that we have handled data types differently, all other sys objects would be fixed automatically.